src/Command/Push/PushCodeCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Acquia\Cli\Command\Push;
+
+use Acquia\Cli\Command\Pull\PullCommandBase;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class PushCodeCommand.
+ */
+class PushCodeCommand extends PullCommandBase {
+
+  protected static $defaultName = 'push:code';
+
+  /**
+   * @var string
+   */
+  protected $dir;
+
+  /**
+   * {inheritdoc}.
+   */
+  protected function configure() {
+    $this->setDescription('Push code from your IDE to a Cloud Platform environment');
+  }
+
+  /**
+   * @param \Symfony\Component\Console\Input\InputInterface $input
+   * @param \Symfony\Component\Console\Output\OutputInterface $output
+   *
+   * @return int 0 if everything went fine, or an exit code
+   * @throws \Exception
+   */
+  protected function execute(InputInterface $input, OutputInterface $output) {
+    $output->writeln("Please use <options=bold>git</> to push code changes upstream.");
```

Comment:
Is this just a placeholder? I think it would be better to have no `push:code` command, rather than a command that does nothing and exits.